src/screens/Learn/CaseStudies/CaseStudyCard.tsx

```diff
@@ -15,8 +15,8 @@ const CaseStudyCard = (props: { cardContent: CaseStudy; url: string }) => {
   const { cardContent, url } = props;
   const { shortTitle, summary, caseStudyId } = cardContent;
   return (
-    <StyledCard>
-      <StyledLink to={`${url}/${caseStudyId}`}>
+    <StyledLink to={`${url}/${caseStudyId}`}>
```

Comment:
We need to add `aria-label` set to the short title here. Screen readers determine how to read links checking the following properties on the link element (in order of precedence, more or less)

- `aria-label`
- `title`
- text of the link (what's inside `<a>` and `</a>`)

As it is right now, the text of the link includes the card title, the alt text of the logo, the summary and "Learn more", so the screen reader will read it as:

> **California Department of Public Health** California Department of Public Health logo California’s CalCAT tool uses CAN data to analyze current COVID spread and project the impact of policy interventions. Learn How

The `aria-label` usually takes precedence over the link text when set, so if we add `aria-label` to the styled link

```tsx
<StyledLink to={} aria-label={shortTitle}>
  {/* ... */}
</StyledLink>
```

the screen reader will announce just the short title:

>  **California Department of Public Health***

In reality, it's a bit more complex than that, [different screen readers announce slightly different things](https://www.deque.com/blog/text-links-practices-screen-readers/), but this will probably be an improvement.

Reply:
on it!